mocked-api/schemasData.ts

```diff
@@ -785,4 +793,80 @@ export const schemasData: { [key: string]: object } = {
       },
     },
   },
+  "azure_eventhubs_source_0.1": {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "azure_namespace_name",
+      "azure_eventhub_name",
+      "azure_shared_access_name",
+      "azure_shared_access_key",
+      "azure_blob_account_name",
+      "azure_blob_access_key",
+      "azure_blob_container_name",
+    ],
+    properties: {
+      azure_namespace_name: {
+        title: "Eventhubs Namespace",
+        description: "The Event Hubs namespace.",
+        type: "string",
+      },
+      azure_eventhub_name: {
+        title: "Eventhubs Name",
+        description: "The Event Hub name.",
+        type: "string",
+      },
+      azure_shared_access_name: {
+        title: "Share Access Name",
+        description: "The Event Hubs SAS key name.",
+        type: "string",
+        "x-group": "credentials",
```

Comment:
@kelvah please can you share a definition about "x-group:gredentials" I thought that it marks that the parameter should be hidden but the value is still visible. See: 
![image](https://user-images.githubusercontent.com/1536609/183027734-a8d04d52-0eb7-4ef8-9295-4afa90220d10.png)

Reply:
What makes a parameter hidden is its actual definition. 
It has to include the `oneOf` property with the double field definition: one with `format="password"` and the other with `type="object"`
You can see an example here:
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/8d96ac75b463442d90c9cd277935847271d1d3e5/mocked-api/schemasData.ts#L205-L221

I'm not sure what the `x-group` property stands for. It's clearly not a json-schema prop. We inherit these from the original kamelet used to implement the connector. See [here for example](https://github.com/apache/camel-kamelets/blob/main/kamelets/azure-eventhubs-source.kamelet.yaml#L53-L67). But we are not actually using them.